resources/views/components/main-menu.blade.php

```diff
@@ -1,75 +1,211 @@
+                        @foreach (config('menu') as $item)
+                            <x-responsive-nav-link href="{{ route($item['route']) }}" :active="request()->routeIs($item['route'])">
+                                {{ __($item['text']) }}
+                            </x-responsive-nav-link>
+                        @endforeach
                     </div>
-                    <div class="py-6">
-                        @if (Route::has('login'))
-                            @auth
-                                <a href="{{ url('/dashboard') }}" class="-mx-3 block rounded-lg px-3 py-2.5 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50">Dashboard</a>
-                            @else
-                                <a href="{{ route('login') }}" class="-mx-3 block rounded-lg px-3 py-2.5 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50">Log in</a>
-                                @if (Route::has('register'))
-                                    <a href="{{ route('register') }}" class="-mx-3 block rounded-lg px-3 py-2.5 text-base font-semibold leading-7 text-gray-900 hover:bg-gray-50">Register</a>
+                    @auth
+                        <div class="py-6">
+                            <div class="flex items-center ">
+                                @if (Laravel\Jetstream\Jetstream::managesProfilePhotos())
+                                    <div class="shrink-0 mr-3">
+                                        <img class="h-10 w-10 rounded-full object-cover"
+                                            src="{{ Auth::user()->profile_photo_url }}" alt="{{ Auth::user()->name }}" />
+                                    </div>
+                                @endif
+
+                                <div>
+                                    <div class="font-medium text-base text-gray-800">{{ Auth::user()->name }}</div>
+                                    <div class="font-medium text-sm text-gray-500">{{ Auth::user()->email }}</div>
+                                </div>
+                            </div>
+                            <div class="mt-3 space-y-1">
+                                <!-- Account Management -->
+                                <x-responsive-nav-link href="{{ route('profile.show') }}" :active="request()->routeIs('profile.show')">
+                                    {{ __('Profile') }}
+                                </x-responsive-nav-link>
+
+                                @if (Laravel\Jetstream\Jetstream::hasApiFeatures())
+                                    <x-responsive-nav-link href="{{ route('api-tokens.index') }}" :active="request()->routeIs('api-tokens.index')">
+                                        {{ __('API Tokens') }}
+                                    </x-responsive-nav-link>
                                 @endif
```

Comment:
The same here. We can remove it as we are not using the API tokens feature

Reply:
Thank you for your suggestion. I agree that we should remove the route since we are not using the API tokens. I will make the necessary changes.